demo/dynamic_embedding/movielens-100k-sync-estimator/export.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+rm -rf ./export_dir
+sh stop.sh
+
+sleep 1
+export TF_CONFIG='{"cluster": {"worker": ["localhost:2222"], "ps": ["localhost:2223"], "chief": ["localhost:2224"]}, "task": {"type": "chief", "index": 0}}'
```

Comment:
Maybe you should modify this file according to you training pipeline?

Reply:
done.